src/hotspot/share/oops/objArrayKlass.hpp

```diff
@@ -89,9 +92,7 @@ class ObjArrayKlass : public ArrayKlass {
   static ObjArrayKlass* allocate_objArray_klass(ClassLoaderData* loader_data,
                                                 int n, Klass* element_klass, TRAPS);
 
-  static ArrayDescription array_layout_selection(Klass* element, ArrayProperties properties);
-
-  virtual objArrayOop allocate_instance(int length, ArrayProperties props, TRAPS);
+ public:
```

Comment:
Is this needed? There's already one at line 60.

Reply:
It's not.  I moved the methods to be where they are in mainline.  Thanks for noticing this.